runtime/common/RuntimeMLIR.cpp

```diff
@@ -572,6 +572,13 @@ std::unique_ptr<MLIRContext> initializeMLIR() {
 
 ExecutionEngine *createQIRJITEngine(ModuleOp &moduleOp,
                                     llvm::StringRef convertTo) {
+  // The "fast" instruction selection compilation algorithm is actually very
+  // slow fast for large quantum circuits. Disable that here. Revisit this
```

Comment:
Just checking that "very slow fast" is intended.

Reply:
Fixed (and elaborated a bit more) in 28254ef28.